### ISSUE TITLE
pip install ask-lang error

### ISSUE THREAD

**Issue by 0x0elliot** [Buscedv/Ask#86]
```
macuser@Macs-MacBook-Air ask % pip install ask-lang

ERROR: Could not find a version that satisfies the requirement ask-lang (from versions: none)
ERROR: No matching distribution found for ask-lang

```

**Comment by Buscedv** [comment]
Ask currently only supports python version 3.9 and later, that's probably what's causing the issue. I have actually been meaning to fix that and make Ask compatible with 3.7 or 3.8 and later. 

So you can either wait until I've released a new version of Ask that supports earlier Python versions (I will probably get that done tomorrow actually), or you can upgrade to Python 3.9 if you don't want to wait.

**Comment by Buscedv** [comment]
Ask is now compatible with Python version 3.8 and later. Try running `pip install ask-lang` again
